src/components/navbar-component/Navbar.jsx

```diff
@@ -28,12 +37,12 @@ const Navbar=()=> {
                 <nav>
                     <ul>
                         <li>
-                            {authState.isUserActive?<button onClick={()=>dispatchAuth({type:"USER_LOGOUT"})}><i class='bx bx-log-out-circle icons'></i></button>:<Link to='/login'><i className='bx bx-log-in-circle icons'></i></Link>}
+                            {authState.isUserActive?<button onClick={()=>logoutHandler(dispatchAuth,dispatchCart)}><i class='bx bx-log-out-circle icons'></i></button>:<Link to='/login'><i className='bx bx-log-in-circle icons'></i></Link>}
                         </li>
                         <li>
                             <div className="badge-wrapper">
-                                <a href="./pages/wishlist_page.html"><i className='bx bx-heart icons'></i></a>
-                                <div className="badge btn-badge">4</div>
+                                <button onClick={()=>{if(authState.isUserActive===true){navigate('/wishlist')}else{navigate('/login')}}}><i className='bx bx-heart icons'></i></button>
```

Comment:
You can separate it into a function for better readability.

Reply:
Got it Sukanya, will make the suggested changes.